src/PrettyPrinting.jl

```diff
@@ -1386,24 +1386,66 @@ end
 #
 ################################################################################
 
-# A non-compiletime preference
-function allow_unicode(flag::Bool)
-  old_flag = is_unicode_allowed()
-  @set_preferences!("unicode" => flag)
-  return old_flag
+@static if VERSION < v"1.8.0-DEV.1465"
+   ALLOW_UNICODE_OVERRIDE_VALUE = nothing
+else
+   ALLOW_UNICODE_OVERRIDE_VALUE::Union{Bool,Nothing} = nothing
 end
```

Comment:
Could also do
```suggestion
const ALLOW_UNICODE_OVERRIDE_VALUE = Ref{Union{Bool,Nothing}}(nothing)
```
and then access it via `ALLOW_UNICODE_OVERRIDE_VALUE[]`

Reply:
`Ref`s are black magic for me, so I would rather keep it, and simplify once we drop 1.6 supprot.

Reply:
A Ref is basically a vector of fixed length 1 :-)